packages/core-ethereum/src/indexer/index.ts

```diff
@@ -96,6 +96,10 @@ class Indexer extends EventEmitter {
     this.chain.subscribeChannelEvents((e) => {
       this.onNewEvents([e])
     })
+    this.chain.subscribeTokenEvents((e) => {
```

Comment:
why do we need to save the events of the token?

Reply:
By letting indexer processing those events, their hash will pass through the transaction-manager, which moves `pending` or `mined` transaction hashes  to `confirmed`.